src/cockpit/389-console/src/lib/database/globalPwp.jsx

```diff
@@ -1427,6 +1515,97 @@ export class GlobalPwPolicy extends React.Component {
                                 {saveBtnName}
                             </Button>
                         </Tab>
+                        <Tab eventKey={4} title={<TabTitleText>Temporary Password Rules</TabTitleText>}>
+                            <Form className="ds-margin-top-xlg ds-margin-left" isHorizontal autoComplete="off">
+                                {this.state.passwordmustchange == false && (
+                                <FormAlert>
+                                    <Alert
+                                    variant="info"
+                                    title='"User Must Change Password After Reset" must be enabled in General Settings to avail of TPR.'
+                                    aria-live="polite"
+                                    isInline
+                                    />
+                                </FormAlert>
+                                 )}
+                                <Grid
+                                    title="Number of times the temporary password can be used to authenticate (passwordTPRMaxUse)."
+                                >
+                                    <GridItem className="ds-label" span={3}>
+                                        Password Max Use
+                                    </GridItem>
+                                    <GridItem span={9}>
+                                        <TextInput
+                                            value={this.state.passwordtprmaxuse}
+                                            type="number"
+                                            id="passwordtprmaxuse"
+                                            aria-describedby="horizontal-form-name-helper"
+                                            name="passwordtprmaxuse"
+                                            isDisabled={!this.state.passwordmustchange}
+                                            onChange={(checked, e) => {
+                                                this.handleTPRChange(e);
+                                            }}
+                                        />
+                                    </GridItem>
+                                </Grid>
+                                {pwSyntaxRows}
+                            </Form>
+                            <Form className="ds-margin-top-xlg ds-margin-left" isHorizontal autoComplete="off">
```

Comment:
change "ds-margin-top-xlg" to "ds-margin-top".  There is too much vertical space between the fields.  Do this for all the Forms please